httpdebug/header.go

```diff
@@ -0,0 +1,7 @@
+package httpdebug
+
+// Header represents a special header that enables debug mode.
+type Header struct {
+	Key   string `toml:"key"`
```

Comment:
This is more straigtforward.
```suggestion
	Header   string `toml:"header"`
```

Value is beter then password :+1:

Reply:
They call it Key+Value in Go http.Header methods:

https://pkg.go.dev/net/http#Header.Add